src/app/modules/react-migration/react-migration.controller.ts

```diff
@@ -240,13 +245,13 @@ export const serveDefault: ControllerHandler = (req, res, next) => {
     showReact = false
     // Delete existing cookie to prevent infinite redirection
     if (req.cookies) {
-      res.clearCookie(config.reactMigration.adminCookieName)
+      res.clearCookie(config.reactMigration.adminCookieNameJan2023)
```

Comment:
Oh... I didn't think you'd change that code.

From a functional perspective, we still have an `adminCookieName` which controls overrides so that should stay the same, just it will get a new value (with the _jan23 suffix) Then we now also have a `oldCookieName` variable that we want to track to delete it.

Reply:
[7ebfd3b](https://github.com/opengovsg/FormSG/pull/5665/commits/7ebfd3b5fead285a25667d4bdce15d019e1253a7)